.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: Format, lint and test
+name: Test
```

Comment:
+1. 
Test makes sense.

Reply:
True. I had simplified it down for the GitHub Status Badges added to README.md.
As part of #245 might change again.